pkg/master/master.go

```diff
@@ -194,6 +199,8 @@ type Master struct {
 	endpointRegistry  endpoint.Registry
 	portalAllocator   service.IPRegistry
 
+	nodePortAllocator *portallocator.PortAllocator
```

Comment:
serviceNodePortAllocator - sorry, "portal" is pretty clear, but "nodePort" is not.